packages/@aws-cdk/aws-appsync/test/integ.graphql-opensearch.ts

```diff
@@ -8,7 +8,7 @@ const app = new cdk.App();
 const stack = new cdk.Stack(app, 'appsync-opensearch');
 const user = new User(stack, 'User');
 const domain = new opensearch.Domain(stack, 'Domain', {
-  version: opensearch.EngineVersion.OPENSEARCH_1_1,
+  version: opensearch.EngineVersion.OPENSEARCH_1_2,
```

Comment:
This is the interesting test. You probably need to run `yarn integ integ.graphql-opensearch.js` to ensure this still works.

Reply:
Did run the integration test. Seems to work fine:
```
************************************************************
buildup done
************************************************************
⌚(18:09:33)📁(aws-cdk/packages/@aws-cdk/aws-appsync)🛠 (chore/add-latest-opensearch-version-1-2)🐍[3.9.9]☁ [DEV]⏱ [1195,99s]
λ yarn integ integ.graphql-opensearch.js
yarn run v1.22.18
$ cdk-integ integ.graphql-opensearch.js
Synthesizing integ.graphql-opensearch.js.
Selected stack: appsync-opensearch
Deploying integ.graphql-opensearch.js...
Deployment succeeded, updating snapshot.
Cleaning up.
Done in 1714.76s.
```

I assume this means the test passed ? (Cluster got deployed in Account / Torn down afterwards).